src/actladder.cpp

```diff
@@ -488,19 +499,61 @@ void actWinningPortal(Entity* my)
 						}
 						strcpy((char*)net_packet->data, "WING");
 						net_packet->data[4] = victory;
+						net_packet->data[5] = cutscene;
 						net_packet->address.host = net_clients[c - 1].host;
 						net_packet->address.port = net_clients[c - 1].port;
-						net_packet->len = 8;
+						net_packet->len = 5;
```

Comment:
should be `net_packet->len = 6;` due to cutscene var

Reply:
Thanks, I was afraid I messed up one of those.